### ISSUE TITLE
compile of release version fails

### ISSUE THREAD

**Issue by MagicAndre1981** [winsiderss/systeminformer#1381]
### Brief description of your issue

I can't compile the Release version when calling [build_release.cmd](https://github.com/winsiderss/systeminformer/blob/master/build/build_release.cmd) 

### Steps to reproduce (optional)

clone repository and run [build_release.cmd](https://github.com/winsiderss/systeminformer/blob/master/build/build_release.cmd) 

### Expected behavior (optional)

that I can compile the app as release 

### Actual behavior (optional)

compile fails with random errors like C1047 (phlib.lib was created with different version of compiler) or missing .h from WDK, compiling the debug version works fine 

I also had to add `-prerelease` to vswhere commandline in several cmds otherwise nothing happens

### Environment (optional)

```shell
Windows 11 2022 22621.x, 22621 SDK/WDK, VS2022 Preview 17.3 Prev 2.1
```

**Comment by dmex** [comment]
> compile fails with random errors like C1047

Have you read the build instructions?
https://github.com/winsiderss/systeminformer/tree/master/build#readme

> TL;DR - run build_thirdparty.cmd

**Comment by MagicAndre1981** [comment]
I did this and it didn't work at time of filling the issue.

Now you added the `-prerelease` parameter and compiling with VS 2022 Preview now works, the random` .h` errors are also gone with the WDK 22621.382, the original 22621.1 WDK had several bugs.
